x-pack/packages/kbn-elastic-assistant/impl/assistant/settings/evaluation_settings/evaluation_settings.tsx

```diff
@@ -95,6 +95,20 @@ export const EvaluationSettings: React.FC = React.memo(() => {
     },
     [setSelectedDatasetOptions]
   );
+  const onDatasetCreateOption = useCallback(
+    (searchValue = []) => {
```

Comment:
nit: This type checks because `searchValue`  is an `any`:

![any](https://github.com/user-attachments/assets/9412ce33-9169-40a8-bd87-14faa8c6bd46)

however at runtime this fails if `searchValue` is `undefined`:

![runtime_error](https://github.com/user-attachments/assets/4b437193-cb11-494e-8d3b-6f2e2853affe)

In practice it doesn't appear to be an issue, because `EuiComboBoxOptionsListProps` defines the  `onCreateOption` prop (where this is used) as:

```
onCreateOption?: (searchValue: string, options: Array<EuiComboBoxOptionOption<T>>) => boolean | void;
```